control-plane-operator/controllers/hostedcontrolplane/ingressoperator/ingressoperator.go

```diff
@@ -116,11 +124,11 @@ func ReconcileDeployment(dep *appsv1.Deployment, params Params, platformType hyp
 			{Name: "KUBECONFIG", Value: "/etc/kubernetes/kubeconfig"},
 			{
 				Name:  "HTTP_PROXY",
-				Value: fmt.Sprintf("socks5://127.0.0.1:%d", kas.KonnectivityServerLocalPort),
+				Value: fmt.Sprintf("http://127.0.0.1:%d", konnectivityProxyPort),
 			},
 			{
 				Name:  "HTTPS_PROXY",
-				Value: fmt.Sprintf("socks5://127.0.0.1:%d", kas.KonnectivityServerLocalPort),
+				Value: fmt.Sprintf("http://127.0.0.1:%d", konnectivityProxyPort),
```

Comment:
do we need to manage HTTPS env var using an https scheme on the value? 
If that so, do we need to handle the TLS certificates in that regard?

Reply:
> do we need to manage HTTPS env var using an https scheme on the value?
No, the client uses the HTTP proxy as a TCP CONNECT tunnel through which the HTTPS traffic is sent.